umbra-js/test/Umbra.test.ts

```diff
@@ -54,6 +58,11 @@ describe('Umbra class', () => {
     sender.connect(ethers.provider);
     receiver = ethers.Wallet.fromMnemonic(mnemonic as string, `${path as string}/${receiverIndex}`);
     receiver.connect(ethers.provider);
+    receivers.push(receiver);
+    for (let i = 4; i < 10; i++) {
```

Comment:
Let's just pick a large number to ensure it never conflicts with the lower indicies, in case we ever bump those or add another account. These integers don't matter, so can just make the loop go from 100 to 106 or something

Reply:
Addressed according the latest comment below: https://github.com/ScopeLift/umbra-protocol/pull/426#discussion_r1158332053